tests/schema/tests.py

```diff
@@ -5719,3 +5738,649 @@ def test_ci_cs_db_collation(self):
             if connection.vendor == "postgresql":
                 with connection.cursor() as cursor:
                     cursor.execute("DROP COLLATION IF EXISTS case_insensitive")
+
+    @unittest.skipUnless(connection.vendor == "postgresql", "PostgreSQL specific")
+    @isolate_apps("schema")
+    def test_serial_to_integer(self):
+        from django.contrib.postgres.fields import SerialField
+
+        self.serial_to_integer_test(SerialField, IntegerField)
```

Comment:
I kind of wish we tested all these in something like `postgres_tests/test_schema` but given this is a pre-existing pattern here import testing against `django.contrib.postgres` this doesn't have to be done in this PR.